composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "event-engine/php-engine",
+  "name": "arnedesmedt/php-engine",
```

Comment:
vendor name should stay event-engine :laughing:

Reply:
Oops :see_no_evil:

Reply:
Thx to look to these MR's!

Reply:
sorry for another long delay :see_no_evil: . As usual, too much topics going on at the same time with [Cody Engine](https://github.com/proophboard/cody-engine) being the most important open source project for me at the moment.

@arnedesmedt I've created a "Core Maintainers" team and invited you. If you want to be able merge PRs yourself, just accept the invitation ;) I trust you to not break the entire functionality of Event Engine ;)

If you like the idea, I'd give the team also access to other event engine packages. Let me know what you think about it.

Reply:
Thx @codeliner I can imagine that maintaining open source projects besides your other work is not that easy :-) 
So I think your solution of creating a team is a good one! Thx to add me and be aware that your work is much appreciated!